unit/util/expr.cpp

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************\
+
+Module: Unit test for expr.h/expr.cpp
+
+Author: Diffblue Ltd
+
+\*******************************************************************/
+
+#include <testing-utils/catch.hpp>
+
+#include <util/expr.h>
+#include <util/std_types.h>
+#include <util/std_expr.h>
+#include <util/arith_tools.h>
+
+
+SCENARIO("bitfield-expr-is-zero", "[core][util][expr]")
+{
+  GIVEN("An exprt representing a bitfield constant of 3")
+  {
+    const exprt bitfield3 = from_integer(mp_integer(3), c_bit_field_typet(signedbv_typet(32), 4));
+
+    THEN("is_zero() should be false")
+    {
+      REQUIRE(!bitfield3.is_zero());
```

Comment:
`REQUIRE_FALSE` is slightly easier to understand

Reply:
Yeah, that's nicer - I'll change.